shared/css/user-menu.scss

```diff
@@ -31,10 +32,19 @@
       
     }
   }
+
+  .user_options, .create_options {
+    left: 0px;
```

Comment:
Double check left-to-right languages.

Reply:
You mean **right-to-left** languages, right? :)

Reply:
Works the same in RTL, the position of the buttons in the header don't change :)